### PR TITLE
Add version tag to readme

### PR DESCRIPTION
**Pull request by hadiidbouk**

### CODE REVIEW

**Review by vinaykharb — APPROVED** [review]
💯

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # fueled plugin
 
 [![fastlane Plugin Badge](https://rawcdn.githack.com/fastlane/fastlane/master/fastlane/assets/plugin-badge.svg)](https://rubygems.org/gems/fastlane-plugin-fueled)
+![Version](https://img.shields.io/badge/version-0.2.6-blue.svg)
```
Comment (humblerookie) [inline README.md:4]: Not so sure about this, shouldn't a plugin badge url be version independent. Let's publish releases from now on with tags and use this url instead

`https://img.shields.io/github/v/release/Fueled/fastlane-plugin-fueled`

**Review by humblerookie — APPROVED** [review]
Consider the suggestion below

File: README.md (hunk 1 above)
Comment (hadiidbouk) [reply README.md:4]: @humblerookie 
💯 you are right.

**Pull request merged**